### ISSUE TITLE
Improve Contrast 

### ISSUE THREAD

**Issue by K8Sewell** [yalelibrary/YUL-DC#331]
# Summary
Lighthouse Audit 2020.07.08 identified the following failing elements - do not have a sufficient contrast ratio: 

### Home Page
* [x]  a#bookmarks_nav.nav-link
* [x]  a.nav-link
* [x]  a.nav-link
* [x]  span.submit-search-text
* [x]  a.btn.btn-primary
* [x]  a.btn.btn-outline-secondary
* [x]  a

### Search Results Page
* [x]  a#bookmarks_nav.nav-link
* [x]  a.nav-link
* [x]  a.nav-link
* [x]  span.submit-search-text
* [x]  a
* [x]  a.page-link

### Object Show Page
* [x]  a#bookmarks_nav.nav-link
* [x]  a.nav-link
* [x]  a.nav-link
* [x]  span.submit-search-text
* [x]  a.catalog_startOverLink.btn.btn-primary
* [x]  a.next
* [x]  a
* [x]  a#emailLink.nav-link
* [x]  a#smsLink.nav-link
* [x]  a#citationLink.nav-link
* [x]  a#manifestLink

**Comment by K8Sewell** [comment]
Wait on this ticket until Yale fonts and colors are imported / added to code base

**Comment by K8Sewell** [comment]
PR up for this ticket - https://github.com/yalelibrary/yul-dc-blacklight/pull/183


# Screenshots

### Home Page
![image.png](https://images.zenhubusercontent.com/5eac63d9b58bd7046578e120/30589dc4-acc1-45df-8e8a-49846bb0a278)

### Search Results Page
![image.png](https://images.zenhubusercontent.com/5eac63d9b58bd7046578e120/3cf00e12-e1bb-48fc-8576-2df0cbf2b1c4)

### Object Show Page
![image.png](https://images.zenhubusercontent.com/5eac63d9b58bd7046578e120/a76d0ffb-1a06-4c90-8abc-11b9a410401c)
